app/assets/javascripts/teleconsultation-fields.js

```diff
@@ -11,3 +17,50 @@ function toggleTeleconsultationFields(checkbox) {
         teleconsultationFields.style.display = 'none';
     }
 }
+
+function addPhoneNumberFields() {
+    // Clones the first phone number fieldset, and resets it to an empty form state.
+    // Uses Date.now() to index new phone numbers
+
+    let elem = document.querySelector('#mo_row_0')
+    let clone = elem.cloneNode(true)
+    let msec = Date.now()
+    let isd_field = clone.querySelector('input[name="facility[teleconsultation_phone_numbers_attributes][0][isd_code]"]')
+    let phone_number_field = clone.querySelector('input[name="facility[teleconsultation_phone_numbers_attributes][0][phone_number]"]')
+    let delete_field = clone.querySelector('input[name="facility[teleconsultation_phone_numbers_attributes][0][_destroy]"]')
+    let button = clone.querySelector('button[name="delete_button"]')
+
+    clone.id = "mo_row_" + msec
+
+    isd_field.id = "facility[teleconsultation_phone_numbers_attributes][" + msec + "][isd_code]"
+    isd_field.name = "facility[teleconsultation_phone_numbers_attributes][" + msec + "][isd_code]"
+    isd_field.removeAttribute('disabled')
+
+    phone_number_field.id = "facility[teleconsultation_phone_numbers_attributes]["+ msec + "][phone_number]"
+    phone_number_field.name = "facility[teleconsultation_phone_numbers_attributes][" + msec + "][phone_number]"
+    phone_number_field.removeAttribute('disabled')
+    phone_number_field.value = ''
+
+    delete_field.id = "facility[teleconsultation_phone_numbers_attributes][" + msec + "][_destroy]"
+    delete_field.name = "facility[teleconsultation_phone_numbers_attributes][" + msec + "][_destroy]"
+    delete_field.setAttribute('value', "false")
+
+
+    button.removeAttribute('disabled')
+
+    $MORows().appendChild(clone)
+}
+
+function deletePhoneNumber(button) {
+    // Sets the _destroy field's value to true
+    // Disables form inputs for the deleted row
```

Comment:
If we can't undo the deletion, then we should just hide the whole row, instead of disabling.

Reply:
I didn't want to do that because the row isn't actually deleted until the form is submitted. 
Don't want people to assume the phone number has been deleted because it disappeared.

Reply:
Great point 👍